ADAL/src/ADUserInformation+Internal.m

```diff
@@ -53,18 +33,20 @@ -(NSString*) property \
 return nil; \
 }
 
-@implementation ADUserInformation (Internal)
+NSString *const ID_TOKEN_SUBJECT = @"sub";
```

Comment:
those constants are now double :)?

Reply:
@oldalton by "double" you mean it is also in `MSIDIdTokenWrapper`?

Reply:
never mind, I didn't notice it was changed from static to extern and I saw two additions :) 
I have a different question now, if they're only used in ADUserInformation.m, why we need to declare them here?

Reply:
It is because we need to share them between internal & public files.